### ISSUE TITLE
Damp does not correctly show ability popups when interacting with aftermath

### ISSUE THREAD

**Issue by SBird1337** [rh-hideout/pokeemerald-expansion#2627]
Corresponding test in #2626

**Comment by DizzyEggg** [comment]
What exactly is wrong here? In what configuration does the issue happen?

**Comment by SBird1337** [comment]
This is failing:
```c
// Bug: The ability popups don't show up.
SINGLE_BATTLE_TEST("Damp prevents damage from aftermath")
{
    KNOWN_FAILING;
    GIVEN {
        ASSUME(gBattleMoves[MOVE_TACKLE].flags & FLAG_MAKES_CONTACT);
        PLAYER(SPECIES_PARAS) { Ability(ABILITY_DAMP); };
        OPPONENT(SPECIES_VOLTORB) { Ability(ABILITY_AFTERMATH); HP(1); };
        OPPONENT(SPECIES_WOBBUFFET);
    } WHEN {
        TURN { MOVE(player, MOVE_TACKLE); SEND_OUT(opponent, 1); }
    } SCENE {
        ABILITY_POPUP(opponent, ABILITY_AFTERMATH);
        ABILITY_POPUP(player, ABILITY_DAMP);
        NONE_OF(HP_BAR(player));
    }
}
```

**Comment by DizzyEggg** [comment]
Fixed in #2645
